test_yaml.jl

```diff
@@ -0,0 +1,53 @@
+using YAML
+using Dolo
+
+data = YAML.load_file("/home/pablo/Programming/econforge/dolo/examples/models/rbc.yaml")
```

Comment:
Should we copy some of the example models into this repo and then get this path using `Pkg.dir("Dolo", "examples", "models", "rbc.yaml")`?

Then these test can actually pass on CI